### ISSUE TITLE
getSupportedCallingCodes method not found

### ISSUE THREAD

**Issue by rif** [nyaruka/phonenumbers#4]
Hey, thank you for the great library!

In the software I am trying to build I was looking for getSupportedCallingCodes that I found in both Java and python versions.

I tried to get them like this:

```go
var supportedCallingCodes []int
for region := range phonenumbers.GetSupportedRegions() {		
	supportedCallingCodes.append(supportedCallingCodes, phonenumbers.GetCountryCodeForRegion(region))
}
```
but in this way I don't get a few codes (e.g. 800,808,883,870,878,881,882,888,979

Did I missed the method or is there another workaround to get all supported calling codes?

Regards,
-rif

**Comment by nicpottier** [comment]
Probably not implemented yet, can you point me to the code on the Java side?

**Comment by rif** [comment]
Sure, her it is: https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L982

**Comment by nicpottier** [comment]
Just following up on this. As I understand it, `800` etc, aren't normal region codes, but they should be found in `GetSupportedGlobalNetworkCallingCodes()` so the union of those two should include the full list you are looking for. If that looks right the we can add a convenience method for it to match the Java library functionality.

**Comment by rif** [comment]
Indeed, the missing ones are found in GetSupportedGlobalNetworkCallingCodes() so yes, a convenience method would be perfect.

**Comment by nicpottier** [comment]
Added in https://github.com/nyaruka/phonenumbers/pull/11

**Comment by nicpottier** [comment]
Released in v1.0.23
